src/main/flight/pid.c

```diff
@@ -420,6 +420,16 @@ void pidController(const pidProfile_t *pidProfile, const rollAndPitchTrims_t *an
     // Dynamic ki component to gradually scale back integration when above windup point
     const float dynKi = MIN((1.0f - motorMixRange) * ITermWindupPointInv, 1.0f);
 
+    // rotate old I to the new coordinate system
+    const float gyroToAngle = dT * 2.0f * 3.14159f / 360.0f;
+    for( int i = 3; i--; ) {
```

Comment:
Spacing around `(` and `)`.
Also the convention is having increment expression as the last element of `for`.

Reply:
Got it. Decrementing to zero is an old habit of mine to save a bit in code size. But I can certainly write the statement in the more verbose conventional manner.

Reply:
I meant `increment` as in `++` or `--`, e.g. `for (init; cond; incr)`
But going backwards also looks a bit unconventional.

Reply:
The nice part about 'for (uint8_t i = n; i--; )' is that it works for unsigned and is shorter than the conventional 'for (int8_t i = n; i>=0; i--)'. But I understand wanting to keep the code-base consistent and unsigned certainly isn't needed here.

Reply:
gcc is quite clever about optimizing loops ... And readability is IMO worth more than saved instruction (It will take some thinking why rotations are performed in descending order)

Also uint8_t is slower with gcc on ARM (at least used to be in 5.x)